common/operations.go

```diff
@@ -7,10 +7,11 @@ import (
 
 const (
 
-	// Install Operation Commands
-	InstallBookInfoCommand = "install_book_info"
-	InstallHTTPBinCommand  = "install_http_bin"
-	InstallImageHubCommand = "install_image_hub"
+	// Operation Commands
+	BookInfoCommand  = "bookinfo"
```

Comment:
Maybe we could replace the word `command`. It's not really a command, but an operation identifier. What about `Operation`, or `Op`? `BookInfoOperation`or `BookInfoOp`. Would look somewhat more consistent, when creating the map, as well as in `switch`statements.

Reply:
It would be ideal if we did not hard code name operations like this. 

Instead, adapters will encode the mechanics of the operation (i.e. `kubectl`) and operation type (`apply`) and operation media (single file, file path to a folder containing manifests, or URL to an endpoint that returns a manifest) externalizing the operation names ("BookInfo"), supported operation verbs (apply, delete) and supported versions (version of BookInfo) as config outside of the go code.

Reply:
We shouldn't have to release a new version of a service mesh adapter in order to support a new sample application.

Reply:
@leecalcote We would have to have a release for a new sample application support because this repository takes an approach of a library, which will be statically compiled along with the adapter program.

Reply:
Understood. As we refactor, I don't know that this is the ideal approach. My sense is that if we persist in this vein, that we're conceding a better architecture for time-to-market (which may be what we need to do). While we are reachitecting, however, now is the time to raise this concern. Let's have a verbal chat and circle back to document here.

Reply:
Good idea to discuss this now when only a few adapters are using the common library. 
Where would this config be stored? In this repo for common operations, and adapter repo for specific operations. Or in a seperate repo, downloaded dynamically? Are there any risks we should consider?

Reply:
In this repo. Placed on the local filesystem of the docker container of the adapter as a yaml file.

Reply:
Ok, unless there are service mesh specific operations, or necessary modifications of common operations (like now for Consul, for instance), I assume?